test/testdata/java_grpc_client_pipeline.baseline

```diff
@@ -1,4 +1 @@
-mkdir -p {OUTPUT}/library-v1-gen-java/src/main/java
-protoc --proto_path=test/fake-repos/gapi-core-proto/src/main/proto/ --proto_path=test/fake-repos/fake-proto --proto_path=MOCK_GRADLE_TASK_OUTPUT --java_out={OUTPUT}/library-v1-gen-java/src/main/java test/fake-repos/fake-proto/fake.proto
```

Comment:
This is not strictly related to your PR, but just noting that the old structure provides a bit more flexibility than the new structure. It would be nice to be able to set the `--services_root` and `--include_path` flags for `gen-api-package` so that this task will work on APIs that are not in the googleapis repo.

I've filed #39 for this; it can be handled in a different PR.